### DISCUSSION TITLE
Display SH1106 blinkt unregelmässig

### DISCUSSION THREAD

**Discussion by maxcat117** [hoylabs/OpenDTU-OnBattery#1157] [Q&A]
Hallo in die Runde,
bin Neuling und habe heute eine OpenDTU und eine OpenDTU OnBattery auf ESP 32 Wroom 38 Pin installiert.
Bei openDTU funktioniert  das Display einwandfrei, bei openDTU OnBattery blinkt das Display und es ist ein dicker Strich in der Anzeige. 
Habe die Generic NodeMCU 38 pin benutzt.
Was mach ich Falsch, wo liegt der Fehler?

Gruß
Hartmut

**Comment by schlimmchen** [comment]
erster rateversuch: du hast einen pin in der pin mapping doppelt vergeben, und der ist auch am display beteiligt.
